src/components/dynamic/admin/dashboards/Toolbar.jsx

```diff
@@ -28,6 +28,19 @@ const Toolbar = ({
   const [toggle, setToggle] = useState(false);
 
   const onClick = (value) => {
+    const selectedWinners = objects.filter(
+      (obj) => obj.selected && obj.status === "winner"
+    );
+    if (value === "qualify" && selectedWinners.length > 0) {
+      setPopup({
+        title: "Status Change Restricted",
+        text: "Changing status from 'winner' to 'qualify' is restricted. You can check the Prize page for more information.",
```

Comment:
what happens when its not qualify, so they went from winner->pending. Also check the some function, it can return a boolean and do exactly what u need here. u rlly dont need a value check cause the only values u can change to will be non winner so it should raise a flag solely based on selectedWinners array